openedx/features/course_experience/README.rst

```diff
@@ -0,0 +1,54 @@
+Course Experience
+-----------------
+
+This directory contains a Django application that provides the overall
+experience for navigating around an Open edX course.
+
+The course experience consists of a number of views:
+
+1. **Course Home**
+
+   The course home page is the landing page for the course. It presents
+   the learner with information necessary to understand the purpose of the
+   course, its content, and its milestones. It includes a "Course Tools"
+   section that provides links to other tools associated with the course.
```

Comment:
I add `(e.g. reviews, updates, bookmarks, etc.)` in this section about Course Tools.